task1.md

```diff
@@ -25,13 +25,37 @@ ansible-galaxy collection install redhat_cop.ee_utilities
 
 Further documentation for those who are interested to learn more see:
 
-- link1
-- link2
-
-### I think we need to add group_vars here actually... and make sure to add the auth.yml file
+- [installing collections using cli](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html#collections)
+- [using collections in AAP](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html#collections)
 
 ## Step 3
 
+Create a file in this folder path group_vars/all/auth.yml
+
+```yaml
+# User may add controller auth creds to this file and encrypt it using `ansible-vault`
+---
+controller_hostname: "{{ groups['automationcontroller'][0] }}"
+controller_username: "{{ controller_user | default('admin') }}"
+controller_password: "{{ controller_pass | default('Password1234!') }}"
+controller_validate_certs: false
+
+ah_host: "{{ groups['automationhub'][0] }}"
+ah_username: "{{ ah_user | default('admin') }}"
```

Comment:
We already set users and pass, do we want to default these, or change it up once we know the lab more. while defaulting in some cases makes sense, this seems like overkill.

Reply:
we can wait to see what the lab is like to make final decision but we can remove the defaults if you want. I wrote it this way so you didn't have to pass that info assuming you set the admin password to Password1234! it would just work